sample/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+import com.cdsap.talaiot.logger.LogTracker
+import com.cdsap.talaiot.publisher.timeline.TimelinePublisher
+
+repositories {
+    jcenter()
+    mavenCentral()
+    google()
+    mavenLocal()
+}
+
+plugins {
+    kotlin("jvm") version "1.3.11"
+    id("talaiot") version "0.4.0-SNAPSHOT"
+}
+
+talaiot {
+    logger = LogTracker.Mode.INFO
+    publishers {
+        customPublisher = TimelinePublisher(gradle)
```

Comment:
why `TimeLinePublisher` is included as `customPubliser`?
maybe is out of the scope of this PR, but would be nice to have it in the  `publisherConfiguration`

Reply:
This is an option for people to include their custom publisher. I'm removing it from the example, I just expect some people to implement a custom publisher that they can't share publicly.